tests/fixtures/controllers/getController.ts

```diff
@@ -40,6 +40,12 @@ export class GetTestController extends Controller {
     modelsArray: new Array<TestSubModel>(),
     numberArray: [1, 2, 3],
     numberValue: 1,
+    object: {
```

Comment:
Btw, I had to change this because there was a typescript error in tests. That's why I added `tsc` to the `test` command in package.json, so errors caused by previous submissions would be caught sooner.